src/components/input/Select.tsx

```diff
@@ -455,6 +459,7 @@ function SelectMain<T>({
           asNativePopover={listboxAsPopover}
           align={alignListbox}
           classes={popoverClasses}
+          onScroll={onPopoverScroll}
```

Comment:
The overflowing and height controls are on the Popover now, so that's the element where we should listen for `onScroll`.